src/main/java/taxi/controller/car/AddCarController.java

```diff
@@ -30,6 +30,6 @@ public void doPost(HttpServletRequest req, HttpServletResponse resp) throws IOEx
         Manufacturer manufacturer = manufacturerService.get(manufacturerId);
         Car car = new Car(model, manufacturer);
         carService.create(car);
-        resp.sendRedirect(req.getContextPath() + "/cars/add");
+        resp.sendRedirect(req.getContextPath() + "/drivers/cars");
```

Comment:
maybe let's not change this? looks not logical to redirect to driver's cars here